### PR TITLE
Updated design of website: one page

### PR DESCRIPTION
**Pull request by lauramorlans**

### CODE REVIEW

File: _includes/contact/contact-form.html [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
+        <label for="companyname">Company</label>
         <input type="text" id="company" name="companyname" placeholder="Your company name..">
       </div>
       <div class="label-wrap">
         <label for="telephone">Phone number</label>
         <input type="text" id="telephone" name="phonenumber" placeholder="Phone number.." required>
       </div>
       <div class="label-wrap">
-      	<label for="message"> Message</label>
+        <label for="message"> Message</label>
```
Comment (thedavidfraser) [inline _includes/contact/contact-form.html]: Space before the Message text. The others have it removed.

File: _includes/footer/footer.html [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+  	<div class="social-media">
+      <a href="https://twitter.com/weareswoop" target="_blank">
+        <img src="assets/assets/twitter-logo.svg" alt="twitter-logo">
+      </a>
+        <a href="https://www.linkedin.com/company/swoopapplications" target="_blank">
+        <img src="assets/assets/linkedin-logo.svg" alt="linkedin-logo">
+      </a>
+    </div>
+    <ul>
+      <li><a id="button" href="#contact" class="button">Book a demo</a></li>
```
Comment (thedavidfraser) [inline _includes/footer/footer.html]: Is this id used anywhere?

File: _includes/header/header.html [hunk 3]
```diff
@@ -1,46 +1,37 @@
...
-        </div>
+      <div class="nav-colateral-logo">
+        <a href="/"><img src="assets/assets/colateral-logo-header.svg" alt="colateral-logo"></a>
+      </div>
+      <div class="nav-menu-1024">
+        <ul>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+          <li><a id="phone" href="tel:+443300563800">+44 (0) 330 056 3800</a></li>
+          <li><a id="button" href="#contact" class="button">Book a demo</a></li>
```
Comment (thedavidfraser) [inline _includes/header/header.html]: The ids for phone and button are duplicated. Are they used? Are they needed?

File: _includes/intros/product-intro.html [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+<div class="product-container">
+  <div class="product-intro">
+    <div class="product-intro-text">
+      <h1> Bringing clarity to in-store
+        <br>
+        <div class="product-intro-text-alt">marketing campaigns.</div>
+      </h1>
+    </div>
+    <div class="subscribe-button">
+      <a id="button" href="#contact" class="button">book a demo</a>
```
Comment (thedavidfraser) [inline _includes/intros/product-intro.html]: And again

**Comment by lauramorlans** [comment]
@thedavidfraser I removed useless id and refactored css without them!

**Review by thedavidfraser — APPROVED** [review]
Thanks

**Pull request merged**